File: Eulergo/Euler5.go
package main

import "fmt"

func main(){
	div := false
	j := 5250
	for !div {
		for i := 11; i < 20; i++ {
			if j % i != 0 {
				div = false
				j++
				break
			} else {
			div = true
			}
		}
	}
	fmt.Println(j)
}

File: Eulerc/Euler5.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(void){
	int div = 0;
	int i = 1;
	while(!div){
		for(int j = 11; j <= 20; j++){
			if(i%j != 0){
				div = 0;
				i++;
				break;
			}
			else {
			div = 1;
			}
		}
	}
	printf("%d\n", i);
}
